### ISSUE TITLE
Must set config cli flag --tls to `true`

### ISSUE THREAD

**Issue by saulshanabrook** [docker/machine#719]
Currently when I run docker with the flags from `docker-machine config dev`
it says:

```bash
$ docker-machine config dev
--tls --tlscacert=/Users/saul/.docker/machine/machines/dev/ca.pem --tlscert=/Users/saul/.docker/machine/machines/dev/cert.pem --tlskey=/Users/saul/.docker/machine/machines/dev/key.pem -H="tcp://192.168.99.100:2376"⏎
$ docker $(docker-machine config dev) ps
flag provided but not defined: --tls --tlscacert
See 'docker --help'.
```

If instead I change it to `--tls=true ...`, then the docker CLI
will be able to pick it up fine.

```bash
$ docker --tls=true --tlscacert=/Users/saul/.docker/machine/machines/dev/ca.pem --tlscert=/Users/saul/.docker/machine/machines/dev/cert.pem --tlskey=/Users/saul/.docker/machine/machines/dev/key.pem -H="tcp://192.168.99.100:2376" -D version
Client version: 1.5.0
Client API version: 1.17
Go version (client): go1.4.1
Git commit (client): a8a31ef
OS/Arch (client): darwin/amd64
Server version: 1.5.0
Server API version: 1.17
Go version (server): go1.4.1
Git commit (server): a8a31ef

$ docker --tls=true --tlscacert=/Users/saul/.docker/machine/machines/dev/ca.pem --tlscert=/Users/saul/.docker/machine/machines/dev/cert.pem --tlskey=/Users/saul/.docker/machine/machines/dev/key.pem -H="tcp://192.168.99.100:2376" version
Client version: 1.5.0
Client API version: 1.17
Go version (client): go1.4.1
Git commit (client): a8a31ef
OS/Arch (client): darwin/amd64
Server version: 1.5.0
Server API version: 1.17
Go version (server): go1.4.1
Git commit (server): a8a31ef

$ uname -a
Darwin Sauls-MacBook-Pro.local 14.1.0 Darwin Kernel Version 14.1.0: Mon Dec 22 23:10:38 PST 2014; root:xnu-2782.10.72~2/RELEASE_X86_64 x86_64
```

**Comment by shawnzhu** [comment]
This was fixed in docker 1.4.0 via docker/docker#9221, it works for me now.

**Comment by saulshanabrook** [comment]
OK I figured it out, it is only when I use fish shell....

```
$ docker (docker-machine config) ps
```

Must be something about how their subcommand works differently.
